### DISCUSSION TITLE
Amazon Api

### DISCUSSION THREAD

**Discussion by Sokol26** [amzn/selling-partner-api-models#3700] [General]
I have a problem connecting to the API, can anyone help?

Data sent:

```
Array ( [http] => Array ( [header] => content-encoding: amz-1.0
content-type: application/json; charset=utf-8 host:
[webservices.amazon.com](http://webservices.amazon.com/) x-amz-date: 20231002T154651Z x-amz-target:
com.amazon.paapi5.v1.ProductAdvertisingAPIv1.SearchItems Authorization:
AWS4-HMAC-SHA256
Credential=avp2rfm1c8/20231002/us-east-1/ProductAdvertisingAPI/aws4_request,SignedHeaders=content-encoding;content-type;host;x-amz-date;x-amz-target,Signature=2e17691674b419279bd3379e61e7f27758cabe1dc47a8d6fac37bc3c07bfb227
[method] => POST [content] =>
{"PartnerType":"Associates","PartnerTag":"kj035-20","Keywords":"Harry","SearchIndex":"All","Resources":["Images.Primary.Small"],"Marketplace":"[www.amazon.com](http://www.amazon.com/)"}
) )
```

Per resource
```

https://webservices.amazon.com /paapi5/searchitems

response: HTTP/1.1 401 Unauthorized

```
test resource

`https://webservices.amazon.com /paapi5/documentation/quick- start/using-curl.html`

works...

**Comment by rugved1991** [comment]
Hi @Sokol26 ,

We have removed the AWS SigV4 Authorization header requirement. Please use the OAuth Token to call the API and test again to see if you are able to connect to the API.
For more information regarding the AWS SigV4 requirement removal, please refer to this [announcement](https://developer-docs.amazon.com/sp-api/changelog/sp-api-will-no-longer-require-aws-iam-or-aws-signature-version-4).

Best,
Rugved
Solutions Architect, SP-API
